--- dyninst-11.0.0/testsuite-11.0.0/CMakeLists.txt
+++ dyninst-11.0.0/testsuite-11.0.0/CMakeLists.txt
@@ -111,7 +111,8 @@
 if(UNIX)
   enable_language(ASM-ATT)
-  if("${DYNINST_PLATFORM}" MATCHES "i386")
-    enable_language(ASM_NASM)
-  endif()
+# nasm/yasm are deprecated
+#  if("${DYNINST_PLATFORM}" MATCHES "i386")
+#    enable_language(ASM_NASM)
+#  endif()
 elseif(WIN32)
   enable_language(ASM_MASM)

